### ISSUE TITLE
[Experiment] Integrate ParallelWaveGAN with TTS.

### ISSUE THREAD

**Issue by erogol** [mozilla/TTS#311]
repo: https://github.com/kan-bayashi/ParallelWaveGAN
my-fork: https://github.com/erogol/ParallelWaveGAN/tree/tts

Initial experiments show that by PWGAN we can achieve faster than real-time on GPU and close real-time on CPU (as the paper suggests)

- [x] Integrate TTS backend to ParallelWaveGAN 
- [x] Train a LJSpeech model
- [x] Initial inference with TTS + PWGAN
- [x] Fine-tune LJSpeech model with TTS mel-spectrograms. 
- [x] Release the PWGAN model

**Comment by richardburleigh** [comment]
Thanks @erogol! I was able to train PWGAN to sound clear in only two days.

I did however have trouble running inference from this TTS implementation.

Any chance we can get an update to the demo server or notebook to work with the new vocoder?

**Comment by pravn** [comment]
@erogol Do you have trained model for libritts-360? I would like to try. I am looking to adapt a vocoder model for cmu arctic, for which I have my own generations. If parallel wavegan trains fast, maybe I can use that. Do you also have accuracy metrics for PWGAN?

**Comment by erogol** [comment]
i don't have it yet sorry

**Comment by lysektomas** [comment]
Hi Erogol, thanks for this great repo!

I have question about "Fine-tune LJSpeech model with TTS mel-spectrograms.".

Did you implement it here? https://github.com/erogol/ParallelWaveGAN/blob/b19ee611ae29441a474a3383f3792efb2e49c432/parallel_wavegan/bin/preprocess_with_ap.py

Because I don't see model loading, or using tacotron2 as a form of generating spectrogram for learning fine tuned PWGAN. Or did I miss something? I thought that for this Fine-tuning you need mel-spectrograms created not from audio (line 120) but as output from TTS synchronized with GT audio and this (TTS Spectrograms and GT audio) is used for training PWGAN from scratch.

**Comment by erogol** [comment]
There is a notebook to extractt specs from TTS model. I use that,

**Comment by erogol** [comment]
This is done but we start to use our native vocoder module.
